app-backend/tile/src/main/resources/application.conf

```diff
@@ -12,6 +19,25 @@ tile-server {
   cache.enabled = ${?TILESERVER_CACHE_ENABLED}
 }
 
+thumbnail-io-dispatcher {
+  type = Dispatcher
+  executor = "thread-pool-executor"
+  thread-pool-executor {
+    fixed-pool-size = 128
```

Comment:
It seems like these thread pools are too large. Is there a reason why they are this large?